Update service_definition_api.md (#14147)
DataDog/documentation: content/en/tracing/faq/service_definition_api.md

Before you begin, you need a [Datadog API and app key][2].

## Enriching an existing APM service 

> If you are using APM to trace your applications today, you would likely start here. When you land on the Service Catalog page the first time, you will see a ‘UNDEFINED’ pill next to all your existing APM services. 

Reviewer: ```suggestion
If you already use APM to trace your applications, add information about those services. Initially, APM monitored services listed on the Service Catalog page have an `UNDEFINED` label on them. 
```

> To add service ownership information like team name, slack channels, source code repos, you can push a YAML file via the POST endpoint below.

Reviewer: ```suggestion
Add service ownership information such as team name, Slack channels, and source code repositories, by pushing a YAML file using the POST endpoint, described below.
```

## Registering a new service without any Datadog telemetry

> You can manage your service ownership information with Service Catalog even if those services are not emitting any kinds of Datadog telemetry. You can specify service ownership, on-call info, and custom tags in YAML files, and the information will be reflected in the Service Catalog UI. 

Reviewer: ```suggestion
You can manage your service ownership information with Service Catalog even if those services are not emitting any of Datadog telemetry (such as APM traces). Specify service ownership, on-call info, and custom tags in YAML files, and the information is reflected in the Service Catalog UI. 
```
